src/dcd_mapping/vrs_map.py

```diff
@@ -675,14 +677,6 @@ def vrs_map(
     :param silent: If true, suppress console output
     :return: A list of mapping results
     """
-    # TODO address this hardcoding, and if we keep it, this should be a score set mapping error message
-    # if metadata.urn == "urn:mavedb:00000072-a-1":
-    #     msg = f"No RefSeq accession is available for {metadata.urn}."
-    #     if not silent:
-    #         click.echo(msg)
-    #     _logger.warning(msg)
-    #     return None
-
```

Comment:
Will this raise a mapping error now?

Reply:
This score set raises a mapping error without the hardcoding now. This was addressed in the first release of the mapper for mavedb, but I hadn't removed the commented out code.